host_tools/fw_update.py

```diff
@@ -65,8 +65,7 @@ def _get_version(self, port):
             ep.flush()
 
             res = ep.send(ver, GET_VERSION_RESPONSE_ID)
-        except Exception as e:
-            print e
+        except:
```

Comment:
Does this not still catch all Exceptions?  Is there any specific exception you want to catch here?